help/CreateImageAndAzureResources.md

```diff
@@ -124,3 +124,22 @@ Generated tool versions and details can be found in related projects:
 - [Go](https://github.com/actions/go-versions)
 - [Node](https://github.com/actions/node-versions)
 - [Boost](https://github.com/actions/boost-versions)
+
+### Post-generation scripts
+Scripts for setting up some specific configuration after packer build procedure are placed in `post-generation` folder. These scripts are intended to apply configuration which cannot be covered during packer build procedure. Scripts are required, since packer build and default agent users are not the same and some specific configuration should be applied after build process under agent's user.
```

Comment:
```suggestion
Extra configuration should be applied after packer build process, because packer build user and default agent's user are not the same. These configuration scripts ar located in the `post-generation` folder.
```